VEDA_OS01/tests/test_transcripts.py

```diff
@@ -1203,6 +1211,10 @@ def test_translations_retrieval(self):
             decode_func=json.loads,
         )
 
+        # Asserts edx-video-pipeline's video status
+        video = Video.objects.get(studio_id=self.video.studio_id)
+        self.assertEqual(video.video_trans_status, transcripts.VideoStatus.TRANSCRIPTION_READY)
```

Comment:
Nothing to check for IN_PROGRESS ?

Reply:
We don't have test coverage for pipeline delivery phase.